tests/unit/utils/test_analytics_utils.py

```diff
@@ -11,16 +11,173 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
-
+import os
 from contextlib import ExitStack as does_not_raise
+from types import TracebackType
+from typing import Any, Dict, Optional, Type
+from uuid import UUID
+
+from pytest_mock import MockFixture
 
+from zenml.config.global_config import GlobalConfiguration
+from zenml.enums import StackComponentType
 from zenml.utils.analytics_utils import (
     AnalyticsEvent,
     get_segment_key,
     track_event,
 )
 
 
+class environment_handler(object):
```

Comment:
Just did a quick scan and saw this, and was wondering if we can also use the `mocker` to patch the os.environ dict like described here: https://docs.python.org/dev/library/unittest.mock.html#unittest.mock.patch.dict

Reply:
My so-called "quick trick" had an even quicker trick. Thanks for the tip. I replaced it and it is working!